app/templates/components/user-dropdown.hbs

```diff
@@ -4,6 +4,7 @@
     seem to be used as hooks for testing only }}
     <li>{{#link-to 'slugged-route' user.username class="slugged-route"}}Your profile{{/link-to}}</li>
     <li>{{#link-to 'settings.profile' class="profile"}}Settings{{/link-to}}</li>
+    <li>{{#link-to 'settings.integrations' class="integrations"}}integrations{{/link-to}}</li>
```

Comment:
Adding it to the menu should be good enough initially, but I'm ok with a different approach.